### ISSUE TITLE
Still prompted for deployer account index even though none available

### ISSUE THREAD

**Issue by derekpierre** [nucypher/nucypher#1207]
We still prompt for deployer account even when there aren't any:

```
derek @ Dereks-MacBook-Pro-2 ~/Documents/Github/repos/forks/david/nucypher
 [13] → nucypher-deploy contracts --poa --provider file:///Users/derek/Documents/Scratch/Geth/test/geth.ipc --etherscan
WARNING: --no-hw-wallet is enabled.
WARNING: --etherscan is enabled. A browser tab will be opened with deployed contracts and TXs as provided by Etherscan.
Using contract registry filepath /Users/derek/Library/Application Support/nucypher/contract_registry.json
Select deployer account [0]: 0
Error: 0 is not in the valid range of 0 to -1.
Select deployer account [0]:
Traceback (most recent call last):
```

and it leads to a weird looking error:
`Error: 0 is not in the valid range of 0 to -1.`

Perhaps we should check that there is at least one available account before prompting.

**Comment by derekpierre** [comment]
Just realized I was running @cygnusv (`shiny` branch) - was this fixed already?

**Comment by derekpierre** [comment]
Same weird error message on master, except that there is no crash traceback:

```
 [3] → nucypher-deploy contracts --poa --provider file:///Users/derek/Documents/Scratch/Geth/test/geth.ipc
WARNING: --no-hw-wallet is enabled.
Using contract registry filepath /Users/derek/Library/Application Support/nucypher/contract_registry.json
Select deployer account [0]: 0
Error: 0 is not in the valid range of 0 to -1.
Select deployer account [0]: 0
Error: 0 is not in the valid range of 0 to -1.
Select deployer account [0]: 0
Error: 0 is not in the valid range of 0 to -1.
Select deployer account [0]: 0
Error: 0 is not in the valid range of 0 to -1.
Select deployer account [0]:
```

**Comment by cygnusv** [comment]
Fixed in #1206

**Comment by derekpierre** [comment]
👍
